ApartApps.md

```diff
@@ -64,6 +64,63 @@ Data specifically intended for the apartment application is stored in the CCT da
 The details of these tables can be viewed and edited in SQL Server Management Studio with the right permissions. 
 Although, a faster method for simply viewing the tables is to open `Gordon360/Models/CCT_DB_Models.edmx` in Visual Studio 2017 and pan the screen around until the box for the desired table is in view. In the edmx, the type and nullability of each attribute of a table can also be viewed by clicking a column name and looking at the Properties table in the bottom right corner of Visual Studio 2017.
 
+#### Example:
+
+AA_Admin
+- AdminID - varchar(10)
+
+| AdminID  |
+|----------|
+|222222222 |
+|333333333 |
+
+AA_ApartmentApplication
+- AprtAppID - int, not nullable
+- DateSubmitted - datetime, nullable
+- DateModified - datetime, not nullable
+- EditorID - varchar(10), not nullable
+
+| AprtAppID | DateSubmitted           | DateModified            | EditorID  |
+|-----------|-------------------------|-------------------------|-----------|
+| 1         | NULL                    | 2021-01-01 12:12:12.111 | 123456789 |
+| 2         | 2021-01-01 12:12:12.111 | 2021-01-01 12:12:12.111 | 111111111 |
+
+AA_ApartmentChoices
+- AprtAppID - int, not nullable
+- Ranking - int, not nullable
+- BLDG_CDE - char(5), not nullable
```

Comment:
To make the hall names compatible with AA_ApartmentHalls,
```suggestion
- HallName - varchar(15), not nullable
```

Reply:
Out of curiosity, what is the difference between `char()` and `varchar()`

Reply:
char is fixed-length while varchar is variable length. char will insert pads if the given string is shorter than the assigned length. varchar doesn't do that.